srcpkgs/promtail/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'promtail'
+pkgname=promtail
+version=2.6.1
+revision=1
+wrksrc="loki-${version}"
+build_style=go
+go_import_path="github.com/grafana/loki"
+go_package="${go_import_path}/clients/cmd/promtail"
+short_desc="Promtail is an agent which ships the contents of local logs to a Grafana Loki instance"
+maintainer="Mateusz Urbanek <murbanek@shanduur.com>"
+license="Apache-2.0"
+homepage="https://grafana.com/oss/loki/"
+distfiles="https://github.com/grafana/loki/archive/v${version}.tar.gz"
+checksum=4b41175e552dd198bb9cae213df3c0d9ca8cacd0b673f79d26419cea7cfb2df7
```

Comment:
```suggestion
checksum=4b41175e552dd198bb9cae213df3c0d9ca8cacd0b673f79d26419cea7cfb2df7
conf_files=/etc/promtail/promtail-local-config.yaml
```

Reply:
Anyway, what's wrong with just make the file named: `/etc/promtail/config.yaml`?

Reply:
@sgn  will adding just `conf_files=/etc/promtail/config.yaml` create empty file? Or do I need:

```diff
-	vcopy clients/cmd/promtail/promtail-local-config.yaml etc/promtail/promtail-local-config.yaml
+	vcopy clients/cmd/promtail/promtail-local-config.yaml etc/promtail/config.yaml
```